addons/web_editor/static/src/js/frontend/loader.js

```diff
@@ -2,21 +2,18 @@
 
 import { getBundle, loadBundle } from "@web/core/assets";
 
-export async function loadLegacyWysiwygAssets(additionnalAssets=[]) {
-    const xmlids = ['web_editor.assets_legacy_wysiwyg', ...additionnalAssets];
+export async function loadLegacyWysiwygAssets(additionnalAssets=[], preAdditionnalAssests=[]) {
+    // Caution! web_editor.assets_legacy_wysiwyg depends of web_editor.assets_wysiwyg
+    const xmlids = [...preAdditionnalAssests, 'web_editor.assets_legacy_wysiwyg', ...additionnalAssets];
```

Comment:
Because website_root_instance was loading web_editor.assets_wysiwyg after web_editor.assets_legacy_wysiwyg. So I had to add a way to specify how to load the asset before web_editor.assets_legacy_wysiwyg

Reply:
only one bundle and use LoadBundle instead